packages/muelu/test/unit_tests/BlockedRepartition.cpp

```diff
@@ -364,7 +364,18 @@ namespace MueLuTests {
       out << "Skip detailed tests. Matrix was not rebalanced" << std::endl;
       return;
     }
-    TEST_EQUALITY(nNumProcsReb, 2);
+    int expectedPartitions=2;
+#if defined(HAVE_MUELU_KOKKOSCORE) && defined(KOKKOS_HAVE_OPENMP)
```

Comment:
`KOKKOS_HAVE_*` macros have been deprecated and will disappear in the next Kokkos update.  Use `KOKKOS_ENABLE_*` instead.  Thanks!

Reply:
Hmm, that would mean finding and fixing all the code references as well (which is where I stole this ;-).